rotkehlchen/data_import/importers/rotki_trades.py

```diff
@@ -0,0 +1,96 @@
+    def _consume_rotki_trades(
+            self,
+            cursor: DBCursor,
+            csv_row: Dict[str, Any],
+            timestamp_format: str = '%Y-%m-%d %H:%M:%S',
+    ) -> None:
+        """Consume Rotki generic trades import CSV file.
+        May raise:
+        - DeserializationError
+        - UnknownAsset
+        - KeyError
+        - DvisionByZero if `amount_bought` is ZERO
+        """
+        fee = Fee(deserialize_asset_amount(csv_row['Fee'])) if csv_row['Fee'] else None  # noqa: E501
+        fee_currency = (
+            symbol_to_asset_or_token(csv_row['Fee Currency'])
+            if csv_row['Fee Currency'] and fee is not None else None
+        )
+        amount_sold = deserialize_asset_amount(csv_row['Sell Amount'])
+        amount_bought = deserialize_asset_amount(csv_row['Buy Amount'])
+        try:
+            location = Location.deserialize(csv_row['Location'])
+        except DeserializationError:
+            location = Location.EXTERNAL
+        trade = Trade(
+            timestamp=deserialize_timestamp_from_date(
+                date=csv_row['Timestamp'],
+                formatstr=timestamp_format,
+                location='Rotki generic trades importer',
+            ),
+            location=location,
+            fee=fee,
+            fee_currency=fee_currency,
+            rate=Price(amount_sold / amount_bought),
+            base_asset=symbol_to_asset_or_token(csv_row['Base Currency']),
+            quote_asset=symbol_to_asset_or_token(csv_row['Quote Currency']),
+            trade_type=TradeType.SELL if csv_row['Type'] == 'Sell' else TradeType.BUY,
+            amount=amount_bought,
+            notes=csv_row['Description'],
+        )
+        self.add_trade(cursor, trade)
+
+    def _import_csv(self, cursor: DBCursor, filepath: Path, **kwargs: Any) -> None:
+        """May raise:
+        - InputError if one of the rows is malformed
+        """
+        with open(filepath, 'r', encoding='utf-8-sig') as csvfile:
+            data = csv.DictReader(csvfile)
+            for row in data:
+                try:
+                    self._consume_rotki_trades(cursor, row, **kwargs)
+                except UnknownAsset as e:
+                    self.db.msg_aggregator.add_warning(
+                        f'During Rotki generic trades CSV import, found action with unknown '
+                        f'asset {e.asset_name}. Ignoring entry',
+                    )
+                    continue
+                except DeserializationError as e:
+                    self.db.msg_aggregator.add_warning(
+                        f'Deserialization error during Rotki generic trades CSV import. '
+                        f'{str(e)}. Ignoring entry',
+                    )
+                    continue
+                except DivisionByZero:
+                    self.db.msg_aggregator.add_warning(
+                        f'During Rotki generic trades import, csv row {str(row)} has zero '
```

Comment:
Please try to keep rotki lowercase

```suggestion
                        f'During rotki generic trades import, csv row {str(row)} has zero '
```